.github/workflows/rust.yml

```diff
@@ -116,7 +157,12 @@ jobs:
         with:
           toolchain: stable
           command: test
-          args: --release
+          args: --release --workspace
+
+      - name: View logs
+        if: always()
+        run: |
+          cat /tmp/*.log
```

Comment:
@kezhenxu94 Is this step correct? I want to perform this step regardless of whether the previous step is successful or not.